### PR TITLE
Add 1.3.0 release page for vscode extension

### PR DESCRIPTION
**Pull request by Shanedell**
@stevedlawrence I believe this what you were referring to in the vote thread right?

### CODE REVIEW

File: site/_vscode/1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+
+released: false
+apache: true
+title: 1.3.0
+date: 2023-4-27
+summary: >
+    omega-edit data editor and intellisense improvements.
```
Comment (tuxji) [inline site/_vscode/1.3.0.md:8]: Should start with a verb and should join the next sentence (which is only a phrase), like:

```
Adds omega-edit data editor and intellisense improvements
along with package upgrades and bug fixes.
```

File: site/_vscode/1.3.0.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    Along with package upgrades and bug fixes.
+
+source-dist:
+    - "apache-daffodil-vscode-1.3.0-src.zip"
+
+binary-dist:
+    - "apache-daffodil-vscode-1.3.0.vsix"
+---
+
+This release contains new features and fixes
```
Comment (tuxji) [inline site/_vscode/1.3.0.md:18]: This release contains new features and fixes listed below.

**Review by mbeckerle — APPROVED** [review]

File: site/_vscode/1.3.0.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+#### New Features & Fixes
+
+* omega-edit Data Editor no longer experimental.
```
Comment (tuxji) [inline site/_vscode/1.3.0.md:22]: * Uses omega-edit data editor (no longer experimental).

File: site/_vscode/1.3.0.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* Many improvements made toward the omega-edit Data Editor
+    * Includes new UI using Svelte.
+    * Moved more of the server handling to omega-edit instead of having in the extension.
+    * Many of the closed issues deal with the Data Editor and its new UI setup.
```
Comment (tuxji) [inline site/_vscode/1.3.0.md:26]: * Makes many improvements in the omega-edit data editor:
    * Includes new UI using Svelte.
    * Moves more of the server handling to omega-edit from the extension.
    * Closes many issues with the data editor and its new UI setup.

File: site/_vscode/1.3.0.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+* Intellisense enhancements.
+* Added support for outputting Infosets to JSON.
+* Suppress the amount of platform conditionals being used.
```
Comment (tuxji) [inline site/_vscode/1.3.0.md:29]: * Makes Intellisense enhancements.
* Adds support for outputting Infosets to JSON.
* Reduces the number of platform conditionals being used.

File: site/_vscode/1.3.0.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+* Now supports Java 8.
+* Relay diagnostics in the "Parse" section of variables.
```
Comment (tuxji) [inline site/_vscode/1.3.0.md:31]: * Relays diagnostics in the "Parse" section of variables.

File: site/_vscode/1.3.0.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+* Allows for running Daffodil's "limited validation".
+* Improve Launch Wizard Classpath manipulation.
+* Remove uncessary dependcies.
+* Debugger can now be supplied values for variables.
+* LICENSE and NOTICE updates to add missing items and remove others since we removed some dependencies.
+* Initial support for generating TDML files.
+* Updated name of extension to "Apache Daffodil™ Extension for Visual Studio Code" from "VS Code extension for Apache Daffodil".
```
Comment (tuxji) [inline site/_vscode/1.3.0.md:38]: * Improves Launch Wizard Classpath manipulation.
* Removes unnecessary dependencies.
* Allows values to be applied to variables in Debugger.
* Updates LICENSE and NOTICE to add missing dependencies and remove some dependencies.
* Adds initial support for generating TDML files.
* Updates name of extension to "Apache Daffodil™ Extension for Visual Studio Code" from "VS Code extension for Apache Daffodil".

**Review by tuxji — APPROVED** [review]
+1 with some proofreading suggestions

**Review by stevedlawrence — APPROVED** [review]
+1 with @tuxji's suggestions

**Pull request merged**